docs/docs/15-concepts.md

```diff
@@ -45,6 +45,8 @@ include one or more:
 
 Freight can therefore be thought of as a sort of meta-artifact. Freight is what
 Kargo seeks to progress from one stage to another.
+For detailed guidance on working with Freight, refer to
+[this guide](https://kargo.akuity.io/how-to-guides/working-with-freight).
```

Comment:
Putting domain names in a link makes it harder to move the site if/when we need to. (And we just recently did, so it's a thing that can/does happen.)

Just link to pages using file paths instead and Docusaurus will work out the rest:

```suggestion
[this guide](./30-how-to-guides/15-working-with-freight.md).
```

Reply:
Also, I'm fine with you adding this here, but doing so may cause people to overlook the section on the `Freight` CRD that is farther down on this page, so consider following up on this PR very soon to merge any useful parts of that section into the "Working with Freight" page. Eventually, the CRD sections of the concepts doc can then be removed, as well discussed yesterday.